cordova-js-src/platform.js

```diff
@@ -56,9 +57,9 @@ module.exports = {
             // OR cordova.require('cordova/platform').activationContext
             // activationContext:{type: actType, args: args};
             var activationHandler = function (e) {
-                var args = e.detail.arguments;
-                var actType = e.detail.type;
-                platform.activationContext = { type: actType, args: args };
+                // Making all the details available as activationContext
+                platform.activationContext = utils.clone(e.detail); 
+                platform.activationContext.args = e.detail.arguments;       /* for backwards compatibility */
```

Comment:
Is it intentional that we pass the actual object in as `args`?
Is that the backwards compatibility part, or is it just that the object has a property `args` instead of `arguments`
If so I would prefer `platform.activationContext.args = platform.activationContext.arguments;`